edb/api/errors.txt

```diff
@@ -128,6 +128,13 @@
 0x_07_01_00_00   AuthenticationError
 
 
+####
+
+0x_08_00_00_00   AvailabilityError
+
+0x_08_00_00_01   PostgresUnavailableError  #SHOULD_RETRY
```

Comment:
Let's rename to `BackendUnavailableError`. Even though we're bound to Postgres, we usually don't leak that in error messages/types.